src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -132,7 +157,14 @@ export const NavBarSchedule: FC<NavBarScheduleProps> = ({ isVisible }) => {
       dayStartUtcSeconds: getUnixTime(dayStart),
       personalEvents: daysEvents.filter((event) => event.isSaved),
     };
-  }, [relatedVenueEvents, userEventIds, selectedDayIndex, getEventLocation]);
+  }, [
+    relatedVenueEvents,
+    userEventIds,
+    selectedDayIndex,
+    getEventLocation,
+    firstDay,
+    hasScheduleStartDate,
```

Comment:
```suggestion
    firstDayOfSchedule,
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef